.circleci/config.yml

```diff
@@ -7,8 +7,8 @@ docker_config_defaults: &docker_config_defaults
   user: jenkins
   aws_auth:
     # This IAM user only allows read-write access to ECR
-    aws_access_key_id: AKIAI43PKLK3PGLUWQMA
-    aws_secret_access_key: ${CIRCLECI_AWS_SECRET_KEY_FOR_ECR_READ_WRITE}
+    aws_access_key_id: AKIAICW6W6GJO7MIE6CQ
+    aws_secret_access_key: ${CIRCLECI_AWS_SECRET_KEY_FOR_ECR_READ_WRITE_V2}
```

Comment:
Why adding a version number?

Reply:
If we change `CIRCLECI_AWS_SECRET_KEY_FOR_ECR_READ_WRITE` directly instead, all current CircleCI jobs will fail, so we probably need to version it.

Reply:
Hmm I think if you don't hardcode aws_access_key_id but also put it as environment variable, then you will not need to add versioning here and also in the future everytime you update the pair you will not need to change this config file.

Reply:
Good idea - going to do that